src/applications/personalization/dashboard-2/components/CTALink.jsx

```diff
@@ -15,7 +16,7 @@ const CTALink = ({ ariaLabel, href, text, onClick, newTab }) => {
       rel={relProp}
       target={targetProp}
       onClick={onClick || undefined}
-      className="vads-u-margin-top--2 vads-u-display--inline-block"
```

Comment:
The top-margin was actually creating problems in some cases, so I removed it from the default styling. These screenshots show that the CTA Links still look good on the Dashboard, though!

<img width="1100" alt="Screen Shot 2021-04-12 at 12 16 53 PM" src="https://user-images.githubusercontent.com/20728956/114449335-7a52be80-9b89-11eb-9aff-51aa0b2094a8.png">

<img width="1099" alt="Screen Shot 2021-04-12 at 12 17 08 PM" src="https://user-images.githubusercontent.com/20728956/114449351-7e7edc00-9b89-11eb-9c02-016c6f92b56d.png">